scripts/cmfgen2tardis.py

```diff
@@ -0,0 +1,66 @@
+import os
+import sys
+import argparse
+import numpy as np
+import pandas as pd
+
+from tardis.atomic import AtomData
+
+parser = argparse.ArgumentParser()
+parser.add_argument('input_path', help='Path to a CMFGEN abundance file')
+parser.add_argument(
+    'output_path', help='Path to store converted TARDIS abundance file')
+args = parser.parse_args()
+
+atomic_dataset = AtomData.from_hdf5()
+
+
+def get_atomic_number(element):
+    index = -1
+    for atomic_no, row in atomic_dataset.atom_data.iterrows():
+        if element in row['name']:
+            index = atomic_no
+            break
+    return index
+
+
+def convert_format(file_path):
+
+    with open(file_path, 'r') as f:
+        for line in f:
+            items = line.split()
+            n = len(items)
+
+            if 'data points' in line:
+                df = pd.DataFrame(columns=np.arange(int(items[n - 1])),
+                                  index=pd.Index([],
+                                                 name='element'),
+                                  dtype=np.float64)
+
+            if 'mass fraction\n' in line:
+                    abundances = []
+                    element_string = items[0]
+                    atomic_no = get_atomic_number(element_string.capitalize())
+                    element_symbol = atomic_dataset.atom_data.loc[atomic_no]['symbol']
+
+                    #Its a Isotope
+                    if n == 4:
+                        element_symbol += items[1]
+
+                    for line in f:
+                        items = line.split()
+                        if items:
+                            abundances.extend(
+                                np.array(items).astype(np.float64))
+                        else:
+                            break
+
+                    df.loc[element_symbol] = abundances
+        return df.transpose()
+
+
+def parse_file():
```

Comment:
@wkerzendorf I think, you are saying to create a `__main__.py` file under scripts, as in the link you sent. If we do , as in the link you sent, we may  be able to do something like this -
```
tardis_scripts cmfgen2tardis --input_file=/path/file --output_file=/tmp/
tardis_scripts get_spectrum --gdb
```
So, first argument tardis_scripts will be fixed, meaning we want to run scripts, second argument will be the script we want to run, and rest will be arguments required for individual scripts.

By my current approach
`cmfgen2tardis --input_file=/path/file --output_file=/tmp/`
will work, and for this we do not require a `__main__.py` file.[Stackoverflow](https://stackoverflow.com/questions/774824/explain-python-entry-points)

So do you want the 1st way or 2nd way ?